english/weed/weed.md

```diff
@@ -1,7 +1,19 @@
-well weed contains THC which will make you high. 
+well weed contains THC which will make you high.
 To get it out and add to your cookies, use a fat to extract it.
-Butter, Milk, cream can be used. 
-Put the weed in the fat in a pan and add weed. 
-Heat it for 30 minutes. 
-Add it to your cookie dough and bake them. 
-See you - Love Pink Unicorn
\ No newline at end of file
+Butter, Milk, cream can be used.
+Put the weed in the fat in a pan and add weed.
+Heat it for 30 minutes.
+Add it to your cookie dough and bake them.
+See you - Love Pink Unicorn
+
+Then you wonder why you're taking advice from a Love Pink Unicorn (after all, these are known to be the most untrustworthy breed of unicorn, far worse, even, than the Grim Charcoal Unicorns of Strawgoh).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.